src/libopensc/card-isoApplet.c

```diff
@@ -67,6 +71,23 @@ static struct sc_card_driver isoApplet_drv =
 	NULL, 0, NULL
 };
 
+static struct isoapplet_supported_ec_curves {
+		struct sc_object_id oid;
+		size_t size;
+		unsigned int min_applet_version;
+} ec_curves[] = {
+	{{{1, 2, 840, 10045, 3, 1, 1, -1}},     192, 0x0000}, /* secp192r1, nistp192, prime192v1, ansiX9p192r1 */
```

Comment:
Drivers and frameworks should use a bigger common code base. Currently, OIDs/names/parameters can already be found in https://github.com/OpenSC/OpenSC/blob/master/src/libopensc/pkcs15-sc-hsm.c#L42, https://github.com/OpenSC/OpenSC/blob/master/src/libopensc/pkcs15-pubkey.c#L1381, https://github.com/OpenSC/OpenSC/blob/master/src/tools/pkcs11-tool.c#L65 and you are now introducing yet an other code duplication in `card-isoApplet.c`. Yes, I am suggesting to merge this code.

Also, supporting named curves and explicit parameters at the same time is not too hard...